sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -30,6 +31,39 @@
 # This module is experimental. No backwards-compatibility guarantees.
 
 
+class StatusServer(object):
+
+  def start(self, STATUS_HTTP_PORT=0, started_callback=None):
+    """Executes the serving loop for the status server."""
+
+    class StatusHttpHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+      """HTTP handler for serving stacktraces of all worker threads."""
+
+      def do_GET(self):  # pylint: disable=invalid-name
+        """Return /threadz information for any GET request."""
```

Comment:
How about we use `stacktraces` instead of `/threadz`?

Reply:
Sure!